pkg/private/install.py.tpl

```diff
@@ -171,6 +173,9 @@ class NativeInstaller(object):
 
     def do_the_thing(self):
         logging.info("Installing to %s", self.destdir)
+        if self.wipe_destdir:
+            logging.debug("RM %s", self.destdir)
+            shutil.rmtree(self.destdir, ignore_errors=True)
```

Comment:
This will delete the target directory, which will mean that we recreate it, so the perms will change. Unless someone has set that explicitly, this may be unexpected if people had hand created a landing spot. Not that this is wrong, but maybe it deservces some comment in the docs.

Reply:
Done